addons/mail/static/src/model/record_internal.js

```diff
@@ -84,17 +79,15 @@ export class RecordInternal {
             // - 'many' fields => RecordList
             // record[name]?.[0] is ONE_SYM or MANY_SYM
             const recordList = new RecordList();
-            Object.assign(recordList, {
-                [IS_RECORD_LIST_SYM]: true,
-                [ATTR_SYM]: Model._.fieldsAttr.get(fieldName),
-                [ONE_SYM]: Model._.fieldsOne.get(fieldName),
-                [MANY_SYM]: Model._.fieldsMany.get(fieldName),
+            Object.assign(recordList._, {
                 name: fieldName,
                 owner: record,
+            });
+            Object.assign(recordList, {
                 _raw: recordList,
```

Comment:
Shouldn't `_raw` be an internal one too?

Reply:
Probably yes. Everything that doesn't need to be reactive can be moved in internal.
I haven't put `_raw` in internal object for record yet